templates/optimizers.yaml

```diff
@@ -15,6 +15,12 @@ spec:
             path: /app/initial_parameters.json
           - name: qubit-operator
             path: /app/qubitop.json
+          - name: noise-model
```

Comment:
I don't like it, cause it makes these templates slowly grow until they'll have millions of params as they used to in zmachine.
But I hope it'll be fixed in new workflows.

Reply:
Agreed - I don't really see a way around it though. At least not currently